packages/docusaurus-utils/src/__tests__/markdownLinks.test.ts

```diff
@@ -277,10 +277,97 @@ The following operations are defined for [URI]s:
         fileString: `
 [doc a](./doc%20a.md)
 [doc a](<./doc a.md>)
-[doc a](./doc a.md)
```

Comment:
This was a bug. `[doc a](./doc a.md)` is not a Markdown link without the `<>`.

Reply:
Maybe we should keep it in the test then, to show that it's not processed anymore on purpose?

Reply:
It's replaced below by the "not a markdown link" test